api/src/main/java/org/openmrs/module/ugandaemrreports/reports2019/SetupHCT_TST_Facility2019Report.java

```diff
@@ -189,15 +186,15 @@ public ReportDefinition constructReportDefinition() {
         addAgeAndGender(dsd,"s","tested positive through the Other PITC Clinic",df.getPatientsInAll(patientThroughOtherEntryPoints,patientsTestedThroughVCTApproach,testedPositiveDuringPeriod));
         addAgeAndGender(dsd,"t","tested negative through the Other PITC Clinic",df.getPatientsInAll(patientThroughOtherEntryPoints,patientsTestedThroughVCTApproach,testedNegativeDuringPeriod));
 
-        setupTxNewReport.addIndicator(dsd,"PIPa","PIPa positive females",df.getPatientsInAll(females,PIPS,testedPositiveDuringPeriod,patientsTestedThroughHealthFacility),"");
-        setupTxNewReport.addIndicator(dsd,"PIPb","PIPb positive males",df.getPatientsInAll(males,PIPS,testedPositiveDuringPeriod,patientsTestedThroughHealthFacility),"");
-        setupTxNewReport.addIndicator(dsd,"PIPc","PIPc negative females",df.getPatientsInAll(females,PIPS,testedNegativeDuringPeriod,patientsTestedThroughHealthFacility),"");
-        setupTxNewReport.addIndicator(dsd,"PIPd","PIPd negative males",df.getPatientsInAll(males,PIPS,testedNegativeDuringPeriod,patientsTestedThroughHealthFacility),"");
+        org.openmrs.module.ugandaemrreports.reports.Helper.addIndicator(dsd,"PIPa","PIPa positive females",df.getPatientsInAll(females,PIPS,testedPositiveDuringPeriod,patientsTestedThroughHealthFacility),"");
```

Comment:
I think this  should be just 'Helper' without the package path

Reply:
@dbaluku Done